Makefile

```diff
@@ -24,7 +24,7 @@ tags: tags/emacs tags/vim
 tags/emacs: ## Build Emacs-style ctags file
 	@if type ctags >/dev/null 2>&1; then \
 		if ctags --version | grep >/dev/null 2>&1 "Universal Ctags"; then \
-			ctags -e -R --options=share/zsh.ctags --languages=zsh \
+			ctags -e -R --options=share/zsh.ctags --languages=zsh,zsh3 \
```

Comment:
Why name it `zsh3`? It gives no information about what the difference is. Please rename the `zsh3` to something informative.